src/components/Card/CardContent.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="sb-card__content">
+    <slot></slot>
+    <div v-if="isLoading" class="sb-card__loading">
+      <SbLoading size="x-large" color="primary" />
+    </div>
+  </div>
+</template>
+
+<script>
+import SbLoading from '../Loading'
+export default {
+  name: 'SbCardContent',
+  components: {
+    SbLoading,
+  },
+  inject: ['cardContext'],
```

Comment:
Are there enough component levels that justifies using `provide/inject`?
Can't we just use `props` normally here?

Reply:
Not really since it's injected through slots